test/test.js

```diff
@@ -312,3 +312,33 @@ test('wait for test to end', function (t) {
 		avaTest.pass();
 	}, 1234);
 });
+
+test('promise is rejected with the first assertError', function (t) {
```

Comment:
Not sure why `promise` is mentioned here. It's not really relevant. Should be more like:

> fails with the first assertError

Sem with the below.